narayana-spring-boot-core/src/main/java/me/snowdrop/boot/narayana/core/properties/NarayanaProperties.java

```diff
@@ -115,6 +115,11 @@
      */
     private Map<String, String> dbcp = new HashMap<>();
 
+    /**
+     * XA recovery nodes.
+     */
+    private List<String> xaRecoveryNodes = new ArrayList<>(Arrays.asList("1"));
```

Comment:
Why not just `Arrays.asList("1")`? It doesn't seem to be modified anywhere.

Reply:
My bad, just noticed that it's the same in other properties. Not sure why though, will look into it some other time.